### PR TITLE
Issue #1070 Add centos and b2d alias to config set

### PR DESCRIPTION
**Pull request by praveenkumar**
Addresses issue  #1070

### CODE REVIEW

File: cmd/minishift/cmd/config/config.go [hunk 1]
```diff
@@ -43,7 +43,11 @@ type Setting struct {
 	callbacks   []setFn
 }
 
-var settingsList []Setting
+var (
+	settingsList   []Setting
+	B2dIsoAlias    = "b2d"
```
Comment (gbraad) [inline cmd/minishift/cmd/config/config.go]: why `var` and not `const` ?

File: cmd/minishift/cmd/start.go [hunk 2]
```diff
@@ -362,9 +360,9 @@ func determineIsoUrl(iso string) string {
 	iso = strings.ToLower(iso)
 
 	switch iso {
-	case b2dIsoAlias, "":
+	case startFlags.B2dIsoAlias, "":
```
Comment (gbraad) [inline cmd/minishift/cmd/start.go]: in this case we might also be able to use a internal variable to make `""` refer to a named option such as `DefaultIsoOption` to make it clearer why we have this in the case select.

File: cmd/minishift/cmd/start.go [hunk 3]
```diff
@@ -58,9 +59,6 @@ const (
 	defaultProject = "myproject"
 	defaultUser    = "developer"
 
-	b2dIsoAlias    = "b2d"
-	centOsIsoAlias = "centos"
-
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/start.go]: 👍

File: pkg/minikube/constants/constants.go [hunk 4]
```diff
@@ -35,6 +35,7 @@ const (
 	DefaultCPUS                  = 2
 	DefaultDiskSize              = "20g"
 	UpdateMarkerFileName         = "updated"
+	DefaultISO                   = ""
```
Comment (gbraad) [inline pkg/minikube/constants/constants.go]: The risk we have with this is that a constant seems to need a value... and when they give it one, it fails the intended function. Better to leave it as a local named variable

Comment (LalatenduMohanty) [inline pkg/minikube/constants/constants.go]: This does not look right as B2D is the default ISO. I understand that the default thing is handled in start.go  but the question is if any other code uses this if yes, then we should fix it accordingly and see if we need this at all.

File: cmd/minishift/cmd/start.go [hunk 5]
```diff
@@ -360,11 +358,12 @@ func calculateDiskSizeInMB(humanReadableDiskSize string) int {
 
 func determineIsoUrl(iso string) string {
 	iso = strings.ToLower(iso)
+	defaultISO := ""
```
Comment (hferentschik) [inline cmd/minishift/cmd/start.go]: not sure what you gain here with this variable. I think at least the name of the variable is confusing. It is not _isoNotSpecified_

**Comment by hferentschik** [comment]
Rebased and merged, thanks.

**Pull request closed without merging**